### PR TITLE
PHP: stop using annotations from sslr-squid-bridge

### PR DESCRIPTION
**Pull request by pynicolas**

### CODE REVIEW

File: php-custom-rules/pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 	<properties>
 		<sonar.version>6.7</sonar.version>
-		<sonarphp.version>2.12.1.3018</sonarphp.version>
+		<sonarphp.version>2.13.0.3096</sonarphp.version><!-- To be updated when SonarPHP 2.13 is released -->
```
Comment (vilchik-elena) [inline php-custom-rules/pom.xml]: @pynicolas but why build fails now?

File: php-custom-rules/src/main/java/org/sonar/samples/php/PHPRuleRepository.java [hunk 2]
```diff
@@ -21,25 +21,23 @@
...
-public class PHPRulesDefinition extends PHPCustomRulesDefinition {
-
-  /**
-   * Provide the repository name
-   */
-  @Override
-  public String repositoryName() {
-    return "MyCompany Custom Repository";
-  }
+public class PHPRuleRepository implements RulesDefinition, PHPCustomRuleRepository {
```
Comment (vilchik-elena) [inline php-custom-rules/src/main/java/org/sonar/samples/php/PHPRuleRepository.java]: @pynicolas it's a bit confusing to have `repository` implementing `RulesDefinition` interface

File: php-custom-rules/pom.xml (hunk 1 above)
Comment (pynicolas) [reply php-custom-rules/pom.xml]: 2.13.0.3096 is only a PR build, and it seems that the build of sonar-custom-rules-examples doesn't use repox.sonarsource.com.

File: php-custom-rules/src/main/java/org/sonar/samples/php/PHPRuleRepository.java (hunk 2 above)
Comment (pynicolas) [reply php-custom-rules/src/main/java/org/sonar/samples/php/PHPRuleRepository.java]: Could you please elaborate?

Comment (vilchik-elena) [reply php-custom-rules/src/main/java/org/sonar/samples/php/PHPRuleRepository.java]: @pynicolas how about something generic for this class name, e.g. `PHPCustomRules` or `MyPhpRules`?

**Pull request merged**